packages/@lwc/engine-core/src/3rdparty/snabbdom/types.ts

```diff
@@ -63,7 +63,7 @@ export interface VText extends VNode {
     children: undefined;
     elm: Node | undefined;
     text: string;
-    key: undefined;
+    key: Key | undefined;
```

Comment:
```suggestion
    key: Key;
```
Also, can we now now update VNode interface to `key: Key;` and not override it in VElement and VText